src/types/main.d.ts

```diff
@@ -0,0 +1,90 @@
+declare enum WeekdayNamesLong {
```

Comment:
1. Why you should avoid using enums and the alternative syntax using `as const` and `ObjectValues` type - https://www.youtube.com/watch?v=jjMbPt_H3RQ.
2. You should not put your types in the definitions file - https://www.youtube.com/watch?v=zu-EgnbmcLY